lib/3scale_toolbox/commands/import_command/openapi/update_policies_step.rb

```diff
@@ -0,0 +1,92 @@
+module ThreeScaleToolbox
+  module Commands
+    module ImportCommand
+      module OpenAPI
+        class UpdatePoliciesStep
+          include Step
+
+          ##
+          # Updates Proxy Policies config
+          def call
+            # to be idempotent, we must read first
+            source_policies_settings = service.policies
+            # shallow copy
+            # to detect policies_settings changes, should only be updated setting objects
+            # do not update in-place, otherwise changes will not be detected
+            policies_settings = source_policies_settings.dup
+
+            add_anonymous_access_policy(policies_settings)
+            add_rh_sso_keycloak_role_check_policy(policies_settings)
+
+            return if same_policy_settings?(source_policies_settings, policies_settings)
```

Comment:
This seems a bit weird to extract into a method.

Reply:
Right. Initially deep compare was implemented. Later I changed strategy and only shallow copies and shallow compares are used.